packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -34,6 +34,12 @@ type Props = {|
      */
     menuItems: Array<Item> | Item,
 
+    /**
+     * Closes the Dropdown when an OptionItem is selected, use this
+     * prop if you want single-select OptionItems
+     */
+    singleSelectOption?: boolean,
```

Comment:
What do you think about using `singleSelectEnabled` or `singleSelectionEnabled` for consistency with booleans.

To give you some context, there are a lot of props that don't use this naming, but we are trying to get a more consistent API naming, so it would be helpful to name new props this way :)

Reply:
Could this be even more descriptive of its purpose? Like `closeOnSingleSelect` or `closeOnSingleSelectEnabled`?

While the docs here explain what it does, those docs aren't present at places where this actually gets set and having a more descriptive name will then mean that uses are immediately easier to grok.

Reply:
@jandrade I included `Option` in the name because I wanted to convey that this prop only affected the behavior of OptionItems. @somewhatabstract It doesn’t only close the Dropdown however, it also defines the single select behavior by updating the state of the parent component accordingly

Reply:
If that's the case, I'd suggest renaming this prop to something more generic and familiar for devs that previously have worked with `<SingleSelect/>` and `<MultiSelect/>` components. I understand your point about restricting the behavior with the prop name, but I think that maybe it could be better to define that behavior in the value. This way, there would be a mapping between the Dropdown component and the other variants (SingleSelect, MultiSelect and ActionMenu if needed).

Instead of using a boolean here, you could use a union type with values that could help to describe the behavior you are trying to apply.

Here are some examples to try to illustrate my point a bit better:
e.g. 
a. `selectAs: "singleSelect" | "multiSelect"`
b. `type: "single" | "multi"`
c. `selectionType: "single" | "multi"`

..and making one of those options as default.

This way, this prop could be used later without needing to add a new prop for a given behavior.

Any thoughts?

Reply:
I like an enum for this kind of thing.

Reply:
@jandrade I agree, I’ve updated it in my latest commit to reflect your suggesiton